hugo/layouts/index.html

```diff
@@ -105,14 +105,14 @@ <h3 class="card-title" id="common-name">{{ .Params.subtitle | markdownify }}</h3
     const speciesCards = Array.from(document.querySelectorAll('.scilife-species-card'));
 
     function sortAlphabetically(a, b) {
-        const titleA = $(a).find('#science-name').text();
-        const titleB = $(b).find('#science-name').text();
+        const titleA = a.querySelector('#science-name').textContent;
+        const titleB = b.querySelector('#science-name').textContent;
         return titleA.localeCompare(titleB);
     }
 
     function sortRevAlphabetically(a, b) {
```

Comment:
A common trick to reverse sorting is to negate the original sorting function:
```js
sortRevAlphabetically = a, b => -sortAlphabetically(a, b)
```

Reply:
Thanks for the tip, I hadn't thought of that! Applied in commit 171acb4fb80819c65be5362a561d68c08c8c5518